docs/secure-connections/sauce-connect/faq.md

```diff
@@ -6,10 +6,17 @@ sidebar_label: FAQ
 
 If you're new to Sauce Connect Proxy or troubleshooting, this list of frequently asked questions can help you with what you need to know.
 
+
 ## What outbound ports do I need open for Sauce Connect Proxy?
 
 Sauce Connect can only communicate with Sauce Labs via `port 443`. This is not configurable.
 
+## What is the difference between port:443 and --proxy <host:port>?
+
+`port:443` is HTTPS, is the universal default port for https traffic. You need to open this on your firewall to use Sauce Connect. Any page with a SSL/TLS certificate would be secured using HTTPS.
```

Comment:
I would say, "Port 443 is the the typical default port for HTTPS traffic. Sauce Connect will use port 443 when sending traffic to your network when trying to reach the site(s) under test". 

 But the rest makes sene I think @kimsauce